Add investigate and timeline page (#25889)
DataDog/documentation: content/en/service_management/incident_management/investigate/_index.md

---
title: Investigate Incidents
description: Manage the context and work for an incident
aliases:
- /monitors/incident_management/incident_details
- /service_management/incident_management/incident_details
further_reading:
- link: "/service_management/incident_management/declare"
  tag: "Documentation"
  text: "Declare an Incident"
- link: "/service_management/incident_management/describe"
  tag: "Documentation"
  text: "Describe an Incident"
---

## Overview

{{< img src="/service_management/incidents/investigate/incidents_overview_tab.png" alt="Example incident details view from the Overview tab" style="width:100%;" >}}

> An effective incident investigation involves identifying and categorizing the incident, followed by comprehensive data collection to construct a detailed timeline of events. The Datadog Incident Details page plays an important role in effective incident investigation by providing a centralized platform for real-time monitoring, investigation, remediation, collaboration, and analysis. It integrates dynamic dashboards and interactive timelines, which assist responders in visualizing incident data and patterns. Use the incident details to:

Reviewer: ```suggestion
An effective incident investigation starts with identifying and categorizing the incident, followed by comprehensive data collection to construct a detailed timeline of events. The Datadog Incident Details page helps you investigate incidents by providing a centralized platform for real-time monitoring, investigation, remediation, collaboration, and analysis. It includes dynamic dashboards and interactive timelines which assist responders in visualizing incident data and patterns. Use the incident details to:
```

> - Aggregate and display data in real-time to help teams identify root causes and assess impacts efficiently.   

Reviewer: ```suggestion
- Aggregate and display data in real time to help teams identify root causes and assess impacts efficiently.   
```

> - Communicate, track progress, and coordinate remediation efforts effectively through team collaboration features.   

Reviewer: ```suggestion
- Communicate, track progress, and coordinate remediation efforts by using team collaboration features.   
```

- Pivot between various views to explore affected services and dependencies, ensuring a thorough investigation and resolution.

## Incident details

> Every incident in Datadog has its own Incident Details page where you can manage property fields, signals, tasks, documents, responders, and notifications. The Incident Details page contains a global header for quick access to key actions, while the remaining body of the page is divided into different sections using tabs to group related incident data together.

Reviewer: ```suggestion
Every incident in Datadog has its own Incident Details page where you can manage property fields, signals, tasks, documents, responders, and notifications. The Incident Details page contains a global header for quick access to key actions. The remainder of the page is divided into tabbed sections which group related incident data together.
```

> ### Global Header

Reviewer: ```suggestion
### Global header
```

> The global header provides access to the [Status and Severity][1] selectors, and links to your [Incident Integrations][2]. For more information on how to configure automatic links with every new incident for Slack and Microsoft Teams links, see the [Incident Settings][3] documentation.

Reviewer: ```suggestion
The global header provides access to the [Status and Severity][1] selectors and links to your [Incident Integrations][2]. For more information on how to configure automatic Slack and Microsoft Teams links for every new incident, see the [Incident Settings][3] documentation.
```

> After an incident is resolved, an option appears in the header to generate a postmortem Notebook using a [postmortem template][4]. Configure your postmortem templates in the app; navigate to the [Incident Settings][5] page to predefine the structure and content of your postmortems.

Reviewer: ```suggestion
After an incident is resolved, an option appears in the header to generate a postmortem Notebook using a [postmortem template][4]. To configure your postmortem templates in the app, navigate to the [Incident Settings][5] page and define the structure and content of your postmortems.
```

### Overview tab

> The Overview tab serves as the main page to view an incident's properties and define customer impact. By default, it includes properties such as Root Cause, Services, Teams, Detection Method, and Summary, categorized into the What Happened, Why it Happened, and Attributes sections. 

Reviewer: ```suggestion
The Overview tab serves as the main page to view an incident's properties and define customer impact. By default, it includes properties such as Root Cause, Services, Teams, Detection Method, and Summary. These properties are categorized into the What Happened, Why it Happened, and Attributes sections. 
```

> Add more property fields using `<KEY>:<VALUE>` pairs from Datadog metric tags, or create custom fields through [Incident Settings][6]. Assign values to these properties to improve search and query in the Incident Homepage and Incident Management Analytics. To prioritize critical information, property fields can be reordered and moved under various headings. 

Reviewer: ```suggestion
Add more property fields using `<KEY>:<VALUE>` pairs from Datadog metric tags, or create custom fields through [Incident Settings][6]. Assign values to these properties to improve searches and queries in the Incident Homepage and Incident Management Analytics. To prioritize critical information, you can reorder property fields and move them under various headings. 
```

For customer-facing incidents, specify impact details by adding them in the **Impacts** section:

{{< img src="/service_management/incidents/investigate/incident_details_impacts.png" alt="Your image description" style="width:90%;" >}}

1. Click **Add**.  
2. Specify a start date and time for the impact.  
3. Specify an end date and time for the impact or leave blank if the impact is still ongoing.  
4. Describe the nature of the impact on customers in `Scope of impact`.  
5. Click **Save**.

In addition to housing your property fields, the Overview tab also provides the following at-a-glance summary modules:

| Summary Module | Description |
|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|

> | Condensed Timeline | Displays the times when the incident changes state as well as when impact started and ended for a high-level view of the incident's lifecycle. |

Reviewer: ```suggestion
| Condensed Timeline | Displays the timestamps when incident status changes occurred, as well as when impact started and ended. This provides a high-level view of the incident's lifecycle. |
```

| Latest Notifications| Displays the most recent notification sent for the incident, with quick access to the full list of notifications in the [Notification tab][7].|
| Pending Tasks | Displays the most recent incomplete task, with quick access to the full list of tasks in the Remediation tab. |

> | Responders | Displays the current incident commander and avatars for the remaining responders assigned to the incident. |

Reviewer: ```suggestion
| Responders | Displays the current incident commander as well as avatars for the remaining responders assigned to the incident. |
```

| Recent Timeline Entries | Displays the five most recent entries in the incident timeline, with quick access to see the entire Timeline tab. |

## Additional investigation tools

> After declaring an incident, responders can utilize the Incident Details Page to apply available information to describe and analyze the incident thoroughly. 

Reviewer: ```suggestion
After declaring an incident, responders can use the Incident Details Page to apply available information so that they can describe and analyze the incident thoroughly. 
```

> {{< whatsnext desc="To learn more about other investigative tools, see the following pages:">}}

Reviewer: I would link to this in line 60 instead of using this widget, but there's nothing wrong with the way you've done it.
